back/engines/commercial/project_folders/app/models/project_folders/folder.rb

```diff
@@ -61,36 +62,36 @@ def admin_publication_must_exist
 
     def generate_slug
       slug_service = SlugService.new
-      self.slug ||= slug_service.generate_slug self, self.title_multiloc.values.first
+      self.slug ||= slug_service.generate_slug self, title_multiloc.values.first
     end
 
     def sanitize_description_multiloc
       service = SanitizationService.new
       self.description_multiloc = service.sanitize_multiloc(
-        self.description_multiloc,
-        %i{title alignment list decoration link image video}
+        description_multiloc,
+        %i[title alignment list decoration link image video]
       )
-      self.description_multiloc = service.remove_multiloc_empty_trailing_tags(self.description_multiloc)
-      self.description_multiloc = service.linkify_multiloc(self.description_multiloc)
+      self.description_multiloc = service.remove_multiloc_empty_trailing_tags description_multiloc
+      self.description_multiloc = service.linkify_multiloc description_multiloc
     end
 
     def sanitize_description_preview_multiloc
       service = SanitizationService.new
       self.description_preview_multiloc = service.sanitize_multiloc(
-        self.description_preview_multiloc,
-        %i{decoration link}
+        description_preview_multiloc,
+        %i[decoration link]
       )
-      self.description_preview_multiloc = service.remove_multiloc_empty_trailing_tags(self.description_preview_multiloc)
+      self.description_preview_multiloc = service.remove_multiloc_empty_trailing_tags description_preview_multiloc
     end
 
     def strip_title
-      self.title_multiloc.each do |key, value|
-        self.title_multiloc[key] = value.strip
+      title_multiloc.each do |key, value|
+        title_multiloc[key] = value.strip
       end
     end
 
     def set_admin_publication
-      self.admin_publication_attributes= {} if !self.admin_publication
+      self.admin_publication_attributes = {} if !admin_publication
```

Comment:
```suggestion
      self.admin_publication_attributes = {} unless admin_publication
```

Reply:
Personally, I really don't like `unless`. I find "if not" much more readable than "unless". When formulating these things in human sentences, I think it's more common to say "if not" than "unless" and so I also find it easier to understand code when formulated like this.

But many other developers are also in favour for using "unless", so if everyone else wants to enforce this, I can live with it.

Reply:
I understand and I discussed it with @jinjagit. He said that developers, who are not native English speakers, seem to have trouble with `unless`. In my opinion using `unless` becomes hard to understand when the condition is complex. In this case, it is not. When the condition is complex, probably it has to be abstracted anyway to express intent, for instance by extracting to a variable with a good name, and then the `unless` part becomes simple again. 😃

Reply:
I would clarify: I said "I think it may be harder for non-native English speakers to parse uses of 'unless' _and_ that using `unless` can be difficult to parse in any case, especially when used with `else` statements. I was not implying any correlation of difficulties with non-native English speakers on our team, just theorising that it _might_ add an extra layer of difficulty.